checkout.php

```diff
@@ -80,7 +80,22 @@
                         <div class="media align-items-center">
                             <span class="text-secondary">Total</span>
                             <div class="media-body text-right">
-                                <span class="font-weight-semi-bold">&#8377;<?php echo $total_amount; ?></span>
+                                <span class="font-weight-semi-bold">&#8377;<?php echo $total_amount;
+                                  //ADDITION OPERATION
+                                include 'includes/db_connect.php';
+
```

Comment:
This logic needs to go up where the registration is being inserted into the table.